app/controllers/partners/family_requests_controller.rb

```diff
@@ -31,6 +31,7 @@ def create
       end
 
       create_service = Partners::FamilyRequestCreateService.new(
+        request_type: params[:request_type],
```

Comment:
Don't we know what the request is by the fact that this is the Family Requests Controller? Why are we asking the frontend to pass it in as a parameter?